### DISCUSSION TITLE
Auto hide cursor in other gamemode

### DISCUSSION THREAD

**Discussion by cuhoangdk** [ppy/osu#18515] [Ideas] [answered]
osu!stable can hide the cursor, why can't osu!lazer?

**Comment by smoogipoo** [comment]
It does, but only when the playfield is hovered (in taiko/mania/catch). Which ruleset/mode are you playing?

**Reply by cuhoangdk** [reply]
oh ok, but make it hide in everywhere plz

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/9523
